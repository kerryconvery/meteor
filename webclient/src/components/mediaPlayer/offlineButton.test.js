import React from 'react';
import { shallow } from 'enzyme';
import OfflineButton from './offlineButton';

describe('OfflineButton', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<OfflineButton onClick={() => {}} />);

    expect(wrapper).toMatchSnapshot();
  });
});
