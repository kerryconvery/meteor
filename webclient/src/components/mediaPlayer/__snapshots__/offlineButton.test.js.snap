// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OfflineButton should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <OfflineButton
    onClick={[Function]}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": <div
        className="d-flex justify-content-sm-center"
      >
        <Button
          color="warning"
          onClick={[Function]}
          tag="button"
        >
          Offline
        </Button>
      </div>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <Button
          color="warning"
          onClick={[Function]}
          tag="button"
        >
          Offline
        </Button>,
        "className": "d-flex justify-content-sm-center",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": "Offline",
          "color": "warning",
          "onClick": [Function],
          "tag": "button",
        },
        "ref": null,
        "rendered": "Offline",
        "type": [Function],
      },
      "type": "div",
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": <div
          className="d-flex justify-content-sm-center"
        >
          <Button
            color="warning"
            onClick={[Function]}
            tag="button"
          >
            Offline
          </Button>
        </div>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <Button
            color="warning"
            onClick={[Function]}
            tag="button"
          >
            Offline
          </Button>,
          "className": "d-flex justify-content-sm-center",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "Offline",
            "color": "warning",
            "onClick": [Function],
            "tag": "button",
          },
          "ref": null,
          "rendered": "Offline",
          "type": [Function],
        },
        "type": "div",
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
